hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/key/OMKeyCreateResponse.java

```diff
@@ -91,6 +91,13 @@ public void addToDBBatch(OMMetadataManager omMetadataManager,
         omMetadataManager.getKeyTable(getBucketLayout())
             .putWithBatch(batchOperation, parentKey, parentKeyInfo);
       }
+      
+      // namespace quota changes for parent directory
+      String bucketKey = omMetadataManager.getBucketKey(
+          getOmBucketInfo().getVolumeName(),
+          getOmBucketInfo().getBucketName());
+      omMetadataManager.getBucketTable().putWithBatch(batchOperation,
```

Comment:
Looks like the usedBytes/usedNamespace  update during key/file create was intentionally removed as part of [HDDS-6556](https://issues.apache.org/jira/browse/HDDS-6556) and the update is only upon Key commit. We should perhaps remove this.
cc @captainzmc @errose28

Reply:
@sadanand48 Here we are updating namespace of parent directory of key only, still usedBytes and namespace of the key is done at commitKey only.

But during prepare, we just do validation if quota as required is valid or not which is already present.

Reply:
Got it , thanks!